techniques/index.html

```diff
@@ -74,6 +74,7 @@ <h2>Introduction to Techniques for WCAG 2.2</h2>
 			<h2>Change Log</h2>
 			<p>A list of new, removed or significantly updated techniques:</p>
 			<ol>
+        <li><time datetime="2020-07-15">Nov 16th 2022</time>: Removed F87 Failure of Success Criterion 1.3.1 due to inserting non-decorative content by using :before and :after pseudo-elements and the 'content' property in CSS</li>
```

Comment:
minor nit that it should be `::before` and `::after` rather than the legacy single `:`

but also that the date for the changelog will need to be updated (both since this took a long time to merge, and also because the datetime attribute needs to match the visible date)

Reply:
i just pushed a fix for the incorrect `datetime` attribute